src/main/resources/assets/mythicmetals/lang/ko_kr.json

```diff
@@ -0,0 +1,819 @@
+  "advancement.mythicmetals.midas_gold_sword.description": "업그레이드를 위해선 많은 미다스 골드가 필요한 미다스 골드 검을 만드세요.",
+  "advancement.mythicmetals.midas_gold_sword.title": "로얄 커트러리",
+  "advancement.mythicmetals.mythril.description": "심층암 동굴 상층부에서 흔하게 볼 수 있는 신성한 광석인 미스릴을 찾으세요.",
+  "advancement.mythicmetals.mythril.title": "드워븐 보물",
+  "advancement.mythicmetals.orichalcum.description": "심층암 주변에 흩어져 있는 신성한 광석인 오리칼큠을 찾으세요.",
+  "advancement.mythicmetals.orichalcum.title": "오키쉬 힘",
+  "advancement.mythicmetals.osmium.description": "산 높은 곳에서 오스뮴을 찾으세요.",
+  "advancement.mythicmetals.osmium.title": "풍차는 언제 만들 수 있을까요?",
+  "advancement.mythicmetals.palladium.description": "보통 네더의 용암 바다 아래에 있는 팔라듐을 찾으세요.",
+  "advancement.mythicmetals.palladium.title": "가장 뜨거운 콩",
+  "advancement.mythicmetals.prometheum.description": "정글이나 무성한 동굴 깊숙한 곳에서 프로메튬을 찾으세요.",
+  "advancement.mythicmetals.prometheum.title": "풍성한 채소",
+  "advancement.mythicmetals.runite.description": "벽에 숨겨진 표면 바로 아래에 있는 루나이트 광석을 찾으세요.",
+  "advancement.mythicmetals.runite.title": "제 때 탄생한...",
+  "advancement.mythicmetals.star_platinum.description": "스타라이트, 백금, 카못으로 스타 플래티넘을 단조하세요.",
+  "advancement.mythicmetals.star_platinum.title": "세상에서 가장 특이한 금속",
+  "advancement.mythicmetals.starrite.description": "보통 엔드에서 볼 수 있는 풍부한 크리스털인 스타라이트를 찾으세요.",
+  "advancement.mythicmetals.starrite.title": "떨어진 별의 부산물",
+  "advancement.mythicmetals.steel.description": "철과 망간으로 만드는 간단한 합금인 강철을 단조하세요.",
+  "advancement.mythicmetals.steel.title": "하지만 강철은 더 무거워요...",
+  "advancement.mythicmetals.stormyx.description": "네더랙과 흑암 아래 깊숙한 곳에 숨겨진 스토믹스를 찾으세요.",
+  "advancement.mythicmetals.stormyx.title": "마법의 금속",
+  "advancement.mythicmetals.stormyx_shell.description": "스토믹스 광석에서 희귀하게 떨어지는 스토믹스 껍질을 획득하세요.",
+  "advancement.mythicmetals.stormyx_shell.title": "네더의 다이아몬드",
+  "advancement.mythicmetals.unobtainium.description": "깊숙한 지하에서 흔히 볼 수 있는 언옵테이늄을 획득하세요.",
+  "advancement.mythicmetals.unobtainium.title": "이건 어떻게 얻었었죠?",
+
+  "abilities.mythicmetals.aqua_affinity": "자연스러운 친수성",
+  "abilities.mythicmetals.blast_mining": "폭발 채굴",
+  "abilities.mythicmetals.blast_padding": "폭발 패딩",
+  "abilities.mythicmetals.blast_protection": "자연스러운 폭발로부터 보호",
+  "abilities.mythicmetals.bonus_fortune": "행운 보너스",
+  "abilities.mythicmetals.bonus_looting": "약탈 보너스",
+  "abilities.mythicmetals.carmot_shield": "카못 방패",
+  "abilities.mythicmetals.depth_strider": "자연스러운 물갈퀴",
+  "abilities.mythicmetals.feather_falling": "자연스러운 가벼운 착지",
+  "abilities.mythicmetals.fire_aspect": "자연스러운 발화",
+  "abilities.mythicmetals.knockback": "자연스러운 밀치기",
+  "abilities.mythicmetals.material_tooltip": "희귀한 제작 재료",
+  "abilities.mythicmetals.mending": "재생",
+  "abilities.mythicmetals.projectile_protection": "자연스러운 발사체로부터 보호",
+  "abilities.mythicmetals.respiration": "자연스러운 호흡",
+  "abilities.mythicmetals.smite": "신성",
+  "abilities.mythicmetals.spiked_helm": "가시 박힌 투구",
+  "abilities.mythicmetals.water_protection": "자연스러운 물 보호",
```

Comment:
I translated these to official translation (feather_falling, blast_protection, bonus_fortune, bonus_looting, depth_strider, fire_aspect, knockback, projectile_protection, respiration, water_protection).

I'll change Regrowth, Holy to official translation(수선(mending), 강타(smite)).

Because of unnatural translations('타고난', '천부적인', '자연적인', '자연스러운', '자연의'),
I think it better to except translating 'natural'.
It seems that it's not problem to understand ability mean.
eg) 'Natural Blast Resistance' is to '폭발로부터 보호'

What do you think?

Reply:
That should be fine. It might be better to re-name Regrowth to Self-repair, as it works differently from mending